internal/nanny/manager/manager_test.go

```diff
@@ -258,17 +255,30 @@ func TestGetRoutingInfo(t *testing.T) {
 	addReplica(testData[1])
 
 	// Register two components to start.
-	startComponent("unshardedComponent", false)
-	startComponent("shardedComponent", true)
+	startComponent("sharded", true)
+	startComponent("unsharded", false)
 
-	routingInfo = resolve("")
-	if diff := cmp.Diff(testData[:2], routingInfo.Replicas); diff != "" {
+	routingInfo = resolve("", "unsharded")
+	if diff := cmp.Diff(testData[:2], routingInfo.Routing.Replicas); diff != "" {
 		t.Fatalf("unexpected resolver result (-want,+got):\n%s", diff)
 	}
 	if routingInfo.Version == "" {
 		t.Fatal("no version returned by resolver")
 	}
-	assignment, _ := assigner.FromProto(routingInfo.Assignments[0])
+	if routingInfo.Routing.Assignment != nil {
```

Comment:
In https://github.com/ServiceWeaver/weaver/pull/222, I add a `routing.FormatAssignment` function which might be useful here. It should pretty print assignments in a more readable way than `prototext.Format`. Just a note for the future.

Reply:
ACK, thanks.